test/e2e_scheduler_deprecated/e2e_scheduler_suite_test.go

```diff
@@ -0,0 +1,50 @@
+package e2escheduler
```

Comment:
Don't think we need a new test suite for that

Reply:
Ok, will remove it 👌

Reply:
Removed it.